packages/mdc-floating-label/foundation.ts

```diff
@@ -22,11 +22,11 @@
  */
 
 import {MDCFoundation} from '@material/base/foundation';
-import {SpecificEventListener} from '@material/base/index';
+import {SpecificEventListener} from '@material/base/types';
```

Comment:
if we export the types from `base/index.ts` we don't need to change this.

Reply:
Foundation and adapter files need to use more-specific import paths instead of `/index`, because we might not end up mirroring all `index.ts` files, which would cause broken references when syncing.

However, `index.ts` files _can_ import from other `index.ts` files, since we don't sync them.